packages/forklift-console-plugin/src/modules/Providers/hooks/useStorages.ts

```diff
@@ -0,0 +1,85 @@
+import { useMemo } from 'react';
+
+import {
+  OpenShiftStorageClass,
+  OpenstackVolumeType,
+  OVirtStorageDomain,
+  ProviderType,
+  V1beta1Provider,
+  VSphereDataStore,
+} from '@kubev2v/types';
+import { TypedOvaResource } from '@kubev2v/types/dist/types/provider/ova/TypedResource';
+
+import useProviderInventory from './useProviderInventory';
+
+const apiSlug = (providerType: ProviderType): string => {
```

Comment:
i like the subPath implementation better https://github.com/kubev2v/forklift-console-plugin/pull/877/files#diff-9c58aad2d90a4c6d32c98091332e3f728793a89d8e96b9f67fef175fe1cc0eb6R7